confobo/controllers.py

```diff
@@ -0,0 +1,23 @@
+from confobo.event import Event
+from confobo.user import User
+from confobo import persistence
+
+
+LOWEST_VOTE = 1
+HIGHEST_VOTE = 5
+
+
+def vote(user: User, choice: int, event: Event) -> bool:
```

Comment:
@danaksim could you please connect your vote() method (that handles a command and was created in 4f6022e8e1f31111f9853e24664b074a45d01004) to this controller?